Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1343,7 +1343,7 @@ final class StatusTableViewController: LoopChartsTableViewController {
             hostingController.navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel, target: navigationWrapper, action: #selector(dismissWithAnimation))
             present(navigationWrapper, animated: true)
         } else {
-            let viewModel = CarbEntryViewModel(delegate: loopManager)
+            let viewModel = CarbEntryViewModel(delegate: loopManager, enableManualGlucoseEntry: .recommendManualGlucoseEntry == statusRowMode)
```

Comment:
It's a little indirect to enable the glucose entry view based on `statusRowMode`.  If statusRowMode had a higher priority status available, like bolusing, but we were still in a situation without cgm data, I would expect the manual glucose entry view to still show up.